docs/enable-https.md

```diff
@@ -0,0 +1,59 @@
+# Adding support for HTTPS/SSL
+
+> All the following instructions should be adapted to your personal needs
+
+If your plan to work locally only, first generate your self-signed cert and key:
+
+```bash
+openssl req -x509 -nodes -newkey rsa:2048 -keyout https.key -out https.crt -subj "/CN=localhost" -days 5000
+```
+
+Then copy your cert files on build stage of your Dockerfile:
+
+```Dockerfile
+FROM trafex/php-nginx:latest
+
+# ...
+
+COPY https.crt /etc/nginx/ssl/default.crt
+COPY https.key /etc/nginx/ssl/default.key
+
+# ...
+
+```
+
+Edit your nginx.conf file.
+
+> Check [Nginx configuration](../config/nginx.conf) for more help:
+
+
+```nginx
+server {
+    listen [::]:443 ssl;
+    listen 443 ssl;
+    server_name localhost;
+    root /var/www/html/public;
+
+    ssl_certificate /etc/nginx/ssl/default.crt;
+    ssl_certificate_key /etc/nginx/ssl/default.key;
+
+    # ... the rest here
+}
+```
+
+If you use docker-compose here a example:
```

Comment:
Here **is** an example

Reply:
I just updated my branch but I just don't know how to update the current PR.

Reply:
If you've pushed it to the `patch-1` branch on your repository, the PR should automatically update. I think you've changed it on the `master` branch instead of the `patch-1` branch